Spec/RealtimeClientChannel.swift

```diff
@@ -1079,7 +1079,7 @@ class RealtimeClientChannel: QuickSpec {
 
             describe("detach") {
                 // RTL5a
-                it("if state is INITIALISED, DETACHED or DETACHING nothing is done") {
+                it("if state is INITIALISED and DETACHED nothing is done") {
```

Comment:
What do you mean INITIALISED and DETACHED, it can't be both.  Also, `INITIALIZED` spelling please